extensions/text.js

```diff
@@ -658,6 +686,12 @@
               word[0].toUpperCase() + word.substring(1).toLowerCase();
             return word === titleCased;
           });
+        case CaseParam.CAMELCASE:
+          return string.test(/^[^A-Z\s][^\s]*$/);
```

Comment:
Uncaught TypeError: string.test is not a function

Reply:
Fixed this, it was (pattern).test(string)

Reply:
if you fixed it then push the commit that fixes it, the latest version of the file still errors.

Reply:
Mb, there were two instances of that issue